models/Movie.model.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+
+const movieSchema = new mongoose.Schema({
+    title: String,
+    director: String,
+    stars: Array,
```

Comment:
By using the type Array you have no way to specify the type of each element inside the array, so it is best practices to simply write to type of each element that will be inside the array wrappedin square brackets to identify it will be an array. Like this:
`stars: [ String ]`